### DISCUSSION TITLE
Missing update_payload method

### DISCUSSION THREAD

**Discussion by MrBeeMovie** [qdrant/qdrant-client#634] [Q&A]
As far as I can see there is no update_payload method. Is there any way currently to do an update similar to the update_vectors method for payload? Thank you.

**Comment by joein** [comment]
Hey @MrBeeMovie 

There are [overwrite_payload](https://qdrant.tech/documentation/concepts/payload/?q=overwrite) and [set_payload](https://qdrant.tech/documentation/concepts/payload/?q=set_payload) methods for it

**Reply by MrBeeMovie** [reply]
Ok, thank you. The naming convention for the endpoints are very confusing since they differ for operations on vector, payload, and points as a whole. Sorry, thank you for your time.

**Comment by joein** [comment]
with vectors the operation is straightforward, you can just replace a vector, however with payload you can either extend a payload or overwrite it completely, hence the difference :(

**Reply by MrBeeMovie** [reply]
Got it. Thank you!
